schemas/v1.0/schema.yaml

```diff
@@ -0,0 +1,545 @@
+$id: 'https://spec.openapis.org/arazzo/1.0/schema/2024-08-01'
+$schema: 'https://json-schema.org/draft/2020-12/schema'
+description: |-
+  The description of OpenAPI Initiative Arazzo v1.0.0 documents
+  without schema validation, as defined by https://spec.openapis.org/arazzo/v1.0.0
+type: object
+properties:
+  arazzo:
+    description: The version number of the Arazzo Specification
+    type: string
+    pattern: '^1\.0\.\d+(-.+)?$'
```

Comment:
@frankkilcommins   should this allow `-dev`, `-beta` etc... on the arrazo version?  I was looking at your implementation for Spectral and you have it limited to [0-9]*. 

https://github.com/stoplightio/spectral/pull/2663/files#diff-3df514595919dcfbae1a8420f9bcbd88f7987073e79b965c1e1367bb959f10cfR6

--- 

do we want to align on the proper regex here?  Yours is slightly different than mine too.  Are you using non-capture group for performance reasons or something else?

Reply:
@jeremyfiel allowing `-dev`, `-rc1`, etc. can make sense from an Arrazo project perspective as we're working on enhancements to the specification.

Spectral is more targeted at authors downstream of a published specification version. We would not expect to publish something outside of what's covered by `1\.0(?:\.[0-9]*)?$`. That being said, it was more of a Spectral choice to validate in that way and I followed that convention above.